### ISSUE TITLE
Is MacOS abandoned?

### ISSUE THREAD

**Issue by mackonsti** [efoxbr/megacubo#28]
Hello @efoxbr and thank you for your creativity, effort and maintaining this project alive.

I found out about MegaCubo and wanted to test it on MacOS.

But in your Releases (went back from latest to older ones) I do not see a release for MacOS, no compatible or targeted OS version, there are no PKG or DMG links or builds available (or instructions to compile) and no other information except running the non-notarised application help links you posted.

Is MacOS abandoned ?
Thank you again!

**Comment by efoxbr** [comment]
Hello @mackonsti! I sincerely appreciate your interest in the project and your consideration of my efforts on it.

I'm a PC user and I generated the Mac versions through VMWare, but due to a problem I had on my computer I ended up losing the HDD where this VM was.

As soon as I have time I will redo it, until then, the [last version that is available for macOS is 16.7.4](https://github.com/efoxbr/megacubo/releases/tag/v16.7.4). In recent releases I have added this information to clarify this point.
